specification/conformance/conformance_rules/columns/availabilityzone.json

```
@@ -0,0 +1,99 @@
+{
+  "AvailabilityZone-C-000-C": {
+    "Function": "Composite",
+    "Reference": "AvailabilityZone",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [
+      "AVAILABILITY_ZONE_SUPPORTED"
+    ],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "AvailabilityZone column adheres to the following requirements:",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "AND",
+        "Items": [
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "AvailabilityZone-C-001-M"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "AvailabilityZone-C-002-M"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "AvailabilityZone-C-003-C"
+          }
+        ]
+      },
+      "Condition": {},
+      "Dependencies": [
+        "CostAndUsage-D-003-C",
+        "AvailabilityZone-C-001-M",
+        "AvailabilityZone-C-002-M",
+        "AvailabilityZone-C-003-C"
+      ]
+    }
+  },
+  "AvailabilityZone-C-001-M": {
+    "Function": "Type",
+    "Reference": "AvailabilityZone",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "AvailabilityZone MUST be of type String.",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "TypeString",
+        "ColumnName": "AvailabilityZone"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "AvailabilityZone-C-002-M": {
+    "Function": "Type",
```

ValidationCriteria.Requirement.CheckFunction is Format-related, while the Function is set to Type. 
Noticed the same in the BillingAccountName column.

```suggestion
    "Function": "Format",
```